src/ui/toolbar.js

```diff
@@ -38,7 +38,7 @@ function Toolbar({ settings }) {
         <Icon icon={faPlayCircle} />
       </a>
 
-      <Link to="/settings" title={t('toolbar-upload-settings')}>
+      <Link to="/settings" title={t('toolbar-button-upload-settings')}>
```

Comment:
How did that ever work? Seems to work on studio.opencast.org already. I hope the i18n library does not perform a "closest string search"?

Reply:
It never worked.